### PR TITLE
Direct queries with keys to coordinator

### PR DESCRIPTION
**Pull request by yuce**
This is a PR to https://github.com/pilosa/go-pilosa/tree/import-enhancements

### CODE REVIEW

File: orm.go [hunk 1]
```diff
@@ -112,26 +112,44 @@ func (s *Schema) diff(other *Schema) *Schema {
 	return result
 }
 
+type SerializedQuery struct {
```
Comment (jaffee) [inline orm.go]: does this need to be exported?

File: orm.go [hunk 2]
```diff
@@ -112,26 +112,44 @@ func (s *Schema) diff(other *Schema) *Schema {
 	return result
 }
 
+type serializedQuery struct {
+	Query   string
+	HasKeys bool
+}
+
+func NewSerializedQuery(query string, hasKeys bool) serializedQuery {
```
Comment (jaffee) [inline orm.go]: this function should be unexported as well, I think

File: client.go [hunk 3]
```diff
@@ -50,6 +52,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const PqlVersion = "1.0"
```
Comment (jaffee) [inline client.go]: this should be named PQLVersion according to Go naming conventions

File: orm.go (hunk 2 above)
Comment (yuce) [reply orm.go]: Unexported.

File: client.go (hunk 3 above)
Comment (yuce) [reply client.go]: Renamed

**Pull request merged**